plugins/module_utils/dimensiondata.py

```diff
@@ -19,15 +19,16 @@
 import re
 import traceback
 
-from ansible.module_utils.basic import AnsibleModule, missing_required_lib
+# (TODO: remove AnsibleModule from next line!)
+from ansible.module_utils.basic import AnsibleModule, missing_required_lib  # noqa: F401, pylint: disable=unused-import
 from ansible.module_utils.six.moves import configparser
 from os.path import expanduser
 from uuid import UUID
 
 LIBCLOUD_IMP_ERR = None
 try:
-    from libcloud.common.dimensiondata import API_ENDPOINTS, DimensionDataAPIException, DimensionDataStatus
-    from libcloud.compute.base import Node, NodeLocation
+    from libcloud.common.dimensiondata import API_ENDPOINTS, DimensionDataAPIException, DimensionDataStatus  # noqa: F401, pylint: disable=unused-import
```

Comment:
DimensionDataStatus really looks like an unused import. I couldn't find it anywhere else in plugins/.

Reply:
This is an import in a `try`/`except`, so removing it can have unintended consequences. Maybe it's needed to make sure that a certain version of `libcloud` is present. I cannot judge this, so I think it's best to keep it until someone more qualified looks at it.

Reply:
Same here, I don't know if it's needed for something. All I can say is: It's not used anywhere in plugins/.

Just wanted to mention it in case you've overlooked it.